src/main/scala/io/iohk/praos/App.scala

```diff
@@ -46,15 +48,15 @@ object App extends Logger {
     var genesisHistory: GenesisHistory = GenesisHistoryImpl(
       Map(0 -> Genesis(env.initialStakeDistribution, env.initialNonce))
     )
-    // Run the protocol for 1 epoch
-    (1 to env.epochLength).foreach { _ =>
+    // Run the protocol for 3 epoch
+    (1 to env.epochLength * 3).foreach { _ =>
       val slotInEpoch = slotInEpochCalculator.calculate(env.timeProvider)
       var genesis: Genesis = epochGenesisCalculator.computeGenesisForEpoch(slotInEpoch.epochNumber, genesisHistory).getOrElse(
```

Comment:
Could we replace this with a `val genesis`? AFAIK, this variable is not reassigned

Reply:
I slept on this and you're right